src/deployment/verifyApplicationL2Contracts.js

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable import/no-dynamic-require, no-await-in-loop, no-restricted-syntax, guard-for-in */
+require('dotenv').config();
+const path = require('path');
+const hre = require('hardhat');
+const { expect } = require('chai');
+
+const pathDeployParameters = path.join(__dirname, './deploy_application_parameters.json');
+const pathDeployL1OutputParameters = path.join(__dirname, './deploy_output_l1_applications.json');
+const pathDeployL2OutputParameters = path.join(__dirname, './deploy_output_l2_applications.json');
+
+const deployParameters = require(pathDeployParameters);
+const deployL1OutputParameters = require(pathDeployL1OutputParameters);
+const deployL2OutputParameters = require(pathDeployL2OutputParameters);
+
+async function main() {
```

Comment:
This script was not yet tested? maybe we wanna drop a comment that its untested codde